gitlab/v4/objects/award_emojis.py

```diff
@@ -19,6 +23,42 @@
 ]
 
 
+class GroupEpicAwardEmoji(ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class GroupEpicAwardEmojiManager(NoUpdateMixin, RESTManager):
+    _path = "/groups/{group_id}/epics/{epic_iid}/award_emoji"
+    _obj_cls = GroupEpicAwardEmoji
+    _from_parent_attrs = {"group_id": "group_id", "epic_iid": "iid"}
+    _create_attrs = RequiredOptional(required=("name",))
+
+    def get(
+        self, id: Union[str, int], lazy: bool = False, **kwargs: Any
+    ) -> GroupEpicAwardEmoji:
+        return cast(GroupEpicAwardEmoji, super().get(id=id, lazy=lazy, **kwargs))
+
+
+class GroupEpicNoteAwardEmoji(ObjectDeleteMixin, RESTObject):
+    pass
+
+
+class GroupEpicNoteAwardEmojiManager(NoUpdateMixin, RESTManager):
+    _path = "/groups/{group_id}/epics/{epic_iid}/notes/{note_id}/award_emoji"
+    _obj_cls = GroupEpicNoteAwardEmoji
+    _from_parent_attrs = {
+        "group_id": "group_id",
+        "epic_iid": "epic_iid",
+        "note_id": "id",
+    }
+    _create_attrs = RequiredOptional(required=("name",))
+
+    def get(
+        self, id: Union[str, int], lazy: bool = False, **kwargs: Any
+    ) -> GroupEpicNoteAwardEmoji:
+        return cast(GroupEpicNoteAwardEmoji, super().get(id=id, lazy=lazy, **kwargs))
```

Comment:
Cool. I'm hoping the new `meta` tests helped inform you and what to do to add this. Hopefully the error messages weren't too confusing.

Reply:
I must be honest - I didn't notice them :)
I added these by following the breadcrumbs of the existing notes implementations ;)